crates/ruff_cli/src/commands/check_stdin.rs

```diff
@@ -22,6 +22,15 @@ pub(crate) fn check_stdin(
         if !python_file_at_path(filename, pyproject_config, overrides)? {
             return Ok(Diagnostics::default());
         }
+
+        let lint_settings = &pyproject_config.settings.linter;
```

Comment:
It's a bit annoying that we have to duplicate this check everywhere. It would be nice to have a single CLI logic that handles the traversal of all files and decides if a file is eligible for a specific tool... Another day